am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/base/IntegrationBaseTest.java

```diff
@@ -29,10 +28,7 @@
         databaseHelper = Jdbi.create(db.getJdbcUrl(), db.getUsername(), db.getPassword())
             .installPlugin(new SqlObjectPlugin())
             .onDemand(DatabaseHelperRepository.class);
-    }
 
-    @BeforeAll
-    protected static void cleanupDatabase() {
```

Comment:
truncateTables() method call is not required as  the container is destroyed after tests are run. Every time test is run, it creates a new container for DB.

Reply:
removed truncation

Reply:
👍